File: codility/src/main/java/codility/com/lessons/l06sorting/Triangle.java
package codility.com.lessons.l06sorting;

import java.util.Arrays;

/**
 * 100%
 * @author 08115341622
 */
public class Triangle {
	public int solution(int[] A) {
		long Al[] = new long[A.length];
		for (int i = 0; i < A.length; i++) {
			Al[i] = A[i];
		}
		Arrays.sort(Al);
		for (int i = 0; i < A.length - 2; i++) {
			if (Al[i] + Al[i + 1] > Al[i + 2]) {
				return 1;
			}
		}
		return 0;
	}
}

File: codility_new/triangle.py
# https://app.codility.com/programmers/lessons/6-sorting/triangle/
def solution(A):
    A = sorted(A)
    for i in range(len(A)-2):
        if A[i] > A[i+2] - A[i+1]:
            return 1
    return 0
